api/src/trace/ProxyTracer.ts

```diff
@@ -75,6 +77,9 @@ export class ProxyTracer implements Tracer {
   }
 }
 
+/**
+ * @since 1.0.3
```

Comment:
Reviewer note: Yes, this showed up in a patch version (possibly a mistake): https://github.com/open-telemetry/opentelemetry-js-api/releases/tag/v1.0.3